packages/dev/docs/src/docs.css

```diff
@@ -89,21 +89,21 @@ article {
   }
 }
 
-article h1 {
-  margin-top: 0 !important;
-  font-size: 58px !important;
+article h1.articleHeader {
+  margin-top: 0 !important;;
```

Comment:
extra semi colon

Reply:
still need the important? can we add another class to it instead? that would also increase specificity. or if the article had a class?